Plugin/Framework/Search/Request/AbstractHandler.php

```diff
@@ -98,6 +98,11 @@ abstract class AbstractHandler
      * @var StateAwareCategoryServiceInterface
      */
     protected $categoryService;
+    
+    /**
+     * @var int
+     */
+    protected $pageSize;
```

Comment:
This isn't needed in the parent if it is not referenced.

Reply:
Removed from parent